agent/test-agent-cli/src/test_done.rs

```diff
@@ -0,0 +1,58 @@
+use crate::error::{self, ArchiveSnafu, Result};
+use crate::Client;
+use argh::FromArgs;
+use snafu::ResultExt;
+use std::fs::File;
+use std::time::Duration;
+use tar::Builder;
+use tokio::time::sleep;
+
+#[derive(Debug, FromArgs, PartialEq)]
+#[argh(subcommand, name = "done", description = "Test completed")]
+pub(crate) struct TestDone {}
+
+impl TestDone {
+    pub(crate) async fn run(&self, k8s_client: Client) -> Result<()> {
```

Comment:
What happens if no test results have been sent yet?

Reply:
If no test results have been sent before marking test complete, this will set the state in testsys for the resource as "noTests" and Task state as completed.

Reply:
That's what I'm wondering. Should send test done error if no tests have been sent?

Reply:
How can we track that no test has been sent?? Is there a way to check that from kubernetes client??